### PR TITLE
Feat/config tooltip

### PR DESCRIPTION
**Pull request by pbullhove**
## What does this pull request change?

image
image

### CODE REVIEW

File: packages/dm-core-plugins/src/form/templates/ObjectModelContainedTemplate.tsx [hunk 1]
```diff
@@ -33,6 +33,7 @@ export const ObjectModelContainedTemplate = (
     uiAttribute,
     onOpen
   )
+  console.log(uiAttribute)
```
Comment (eoaksnes) [inline packages/dm-core-plugins/src/form/templates/ObjectModelContainedTemplate.tsx:36]: Please remove console.log :)

**Review by eoaksnes — APPROVED** [review]
👍

File: packages/dm-core-plugins/src/form/widgets/CheckboxWidget.tsx [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+  const { value, readOnly, tooltip } = props
   return (
-    <Checkbox
-      {...props}
-      disabled={readOnly}
-      checked={value !== undefined ? value : false}
-      type='checkbox'
-      data-testid='form-checkbox'
-    />
+    <Tooltip title={tooltip ?? ''}>
```
Comment (soofstad) [inline packages/dm-core-plugins/src/form/widgets/CheckboxWidget.tsx:9]: Are we showing an empty tooltip here?
If so, probably shouldn't

File: packages/dm-core-plugins/src/form/templates/ObjectModelContainedTemplate.tsx (hunk 1 above)
Comment (pbullhove) [reply packages/dm-core-plugins/src/form/templates/ObjectModelContainedTemplate.tsx:36]: thanks!

**Pull request merged**